geomstats/geometry/invariant_metric.py

```diff
@@ -279,13 +282,13 @@ def exp(self, tangent_vec, base_point=None):
             Point in the group equal to the Riemannian exponential
             of tangent_vec at the base point.
         """
+        identity = gs.to_ndarray(self.group.identity, to_ndim=2)
```

Comment:
did you think about the `point_type='matrix'` case?

Reply:
Argh, good point! And the default_point_type is in self.group.default_point_type and not in self.default_point_type so the logic is not handled by the decorator for this case 😭

Reply:
Done.